src/features/movies/MovieDetails/styled.js

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+import bigPoster from "../../../common/images/bigPoster.png";
+
+export const PosterDisplay = styled.div`
+  background-color: ${({ theme }) => theme.color.black};
+`;
+
+export const PosterWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin: 0 auto;
+  padding: 0 16px;
+  max-width: 1368px;
+  height: 769px;
+  background-image: url(${bigPoster});
```

Comment:
When we don't have a poster the background is entirely black, maybe we should added some effects with shadows?

Reply:
When there won't be poster we won't display PosterWrapper at all.